dpnp/tests/third_party/cupy/core_tests/test_ndarray_copy_and_view.py

```diff
@@ -338,7 +338,7 @@ def test_astype_type(self, src_dtype, dst_dtype, order):
         b = astype_without_warning(a, dst_dtype, order=order)
         a_cpu = testing.shaped_arange((2, 3, 4), numpy, src_dtype)
         b_cpu = astype_without_warning(a_cpu, dst_dtype, order=order)
-        assert b.dtype.type == b_cpu.dtype.type
+        assert b.dtype == b_cpu.dtype
```

Comment:
I have changed it to only compare `dtype` because of such cases:

```python
import numpy, dpnp
w=numpy.longlong(3)
a=numpy.ones(10, dtype=numpy.int32)
a.astype(type(w)).dtype
# dtype('int64')
a.astype(type(numpy.longlong(3))).dtype.type
# numpy.longlong

w=dpnp.longlong(3)
a=dpnp.ones(10, dtype=dpnp.int32)
a.astype(type(w)).dtype
# dtype('int64')
a.astype(type(w)).dtype.type
# numpy.int64
```

This is probably because `dpctl.tensor` does not have `dpt.longlong`.